Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Migration/Tools/MigrationWindow.cs

```diff
@@ -0,0 +1,417 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using UnityEngine.SceneManagement;
+using UnityEditor.SceneManagement;
+
+using Object = UnityEngine.Object;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Utilities
+{
+    /// <summary>
+    /// This tool allows the migration of game object obsolete components into up-to-date versions.
+    /// Requires Implementation of IMigrationhandler.
+    /// </summary>
+    public class MigrationWindow : EditorWindow
+    {
+        private enum ToolbarOption 
+        { 
+            Objects = 0, 
+            Scenes = 1, 
+            Project = 2 
+        };
+
+        private static readonly string[] toolbarTitles =
+{
+            "Objects",
+            "Scenes",
+            "Full Project"
+        };
+
+        private ToolbarOption selectedToolbar = ToolbarOption.Objects;
+        private Vector2 scrollPosition = Vector2.zero;
+
+        private const string DependencyWindowURL = "https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Tools/MigrationWindow.html";
+        private const string WindowTitle = "Migration Window";
+        private const string WindowDescription = "This tool allows the migration of obsolete components into up-to-date versions. Component-specific implementation of the MigrationHandler Interface is required.";
```

Comment:
>Component-specific implementation of the MigrationHandler Interface is required [](start = 130, length = 79)

If the window is intended by people who are using it to migrate their stuff, and not so much for folks to write their own migration handlers, then this part of the text is a bit, maybe misleading is the term. 

i.e. as a user of this, does this sentence mean I have to implement component specific IMigrationHandler code? Or do we actually want to communicate that only  a subset of things that have migration handler implementations will actually be migrated? Or we maybe we just want to say "hey check out the docs for the set of things that are migrated by default?"

Reply:
Maybe we remove the message from documentation and window and only leave it in the class summary?

Reply:
agree on that - having it in the class description is enough :)